src/services/expenses/expenses.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+const { v4: uuid } = require('uuid');
+
+const { db } = require('../../db');
+
+const { validateExpenseData } = require('./validation');
+
+const resetStore = async () => {
```

Comment:
Ok, got it. But better to use the migrations concept.

Reply:
Not sure how it should look like
If the table with such a name exists we can check if the schema is correct and if the table is not empty - migrate data to fix the schema if it's needed.
But it's the very first version yet, and there can't exist an "outdated" version